Java Migration/README.md

```diff
@@ -0,0 +1,27 @@
+## Steps for migrating to Azul Zulu OpenJDK from Oracle JRE
+
+Search feature of Azure DevOps Server uses Elasticsearch which depends on Java Runtime Environment (JRE). Up until now, Oracle JRE was the supported version of JRE.  Given the updates to Oracle Java licensing terms which now requires one to buy a subscription to continue to get updates to JRE for commercial use, Azure DevOps Server Search will support Azul Zulu OpenJDK so that our users have an option to choose between Azul Zulu OpenJDK and Oracle JRE, a decision they can make based on their business needs.
+
+Below diagram captures the overall flow of steps invloved in migrating to Azul Zulu OpenJDK from Oracle JRE. 
+![Migration Flow](https://github.com/msftazdev/Code-Search/blob/msftazdev-patch-1/Java%20Migration/flow1.png)
+
+Based on the version of TFS that you are using, please select an appropriate option below to proceed. 
+
+[TFS 2017 RTW](https://github.com/msftazdev/Code-Search/blob/msftazdev-patch-1/Java%20Migration/TFS_2017%20RTW.md)
```

Comment:
The links should be relative to the repo instead of absolute to your fork. EX:
```md
[TFS 2017 RTM](TFS_2017%20RTW.md)
```

Reply:
fixed them